blog: improve the xata sveltekit blog (#207)
xataio/mdx-blog: rag-vector-sveltekit-xata.mdx

Next, proceed to add another column name as `embedding`. This column is responsible for storing the vector embedding generated via OpenAI as `vector` type.

> Among multiple [embedding models offered by OpenAI](https://platform.openai.com/docs/guides/embeddings), the `text-embedding-3-small` model is one of the newest and performant embedding model. The length of the vector embedding generated via the `text-embedding-3-small` model is 1536.

Reviewer: ```suggestion
Among multiple [embedding models offered by OpenAI](https://platform.openai.com/docs/guides/embeddings), the `text-embedding-3-small` model is one of the newest and performant embedding models. The length of the vector embedding generated via the `text-embedding-3-small` model is 1536.
```

xata init --db https://Rishi-Raj-Jain-s-workspace-80514q.us-east-1.xata.sh/db/rag-chatbot
```

> Choose the following answers to one-time setup questions by Xata CLI to integrate Xata with SvelteKit:

Reviewer: ```suggestion
Use the following answers to the Xata CLI one-time setup question prompts to integrate Xata with SvelteKit:
```

- `Yes` when prompted to add `.env` to `.gitignore`.
- `TypeScript` when prompted to select the language to generate code and types from your Xata database.

> - `src/xata.server.ts` when prompted to enter the output path for the generated code by Xata CLI.

Reviewer: ```suggestion
- `src/xata.server.ts` when prompted to enter the output path for the generated code by the Xata CLI.
```
